src/python/botan3.py

```diff
@@ -2138,3 +2149,21 @@ def zfec_decode(k, n, indexes, inputs):
         c_size_t(k), c_size_t(n), c_indexes, c_inputs, c_size_t(share_size), c_outputs
     )
     return [output.raw for output in outputs]
+
+#
+# TPM2
+#
+class TPM2Context:
+    def __init__(self, tcti_nameconf: str = None) -> None:
+        tcti = c_char_p(0 if not tcti_nameconf else tcti_nameconf.encode("utf-8"))
```

Comment:
Other locations (for instance the constructor of `RandomNumberGenerator`) use a `_ctype_str()` helper to transform a string object into a (presumable) `const char*`. Perhaps we should use that here as well? Let's look into it once more...